coords_transformer/coords_calculation.py

```diff
@@ -0,0 +1,21 @@
+from coords_transformer.constants import L, ZOOM
+import math
+
```

Comment:
[nits]
I recommend you should add some references so that people can check whether this calculation is correct or not.
https://www.trail-note.net/tech/coordinate/

Reply:
@tsurutan The reference is in the conftest.py 
Should I move that here or add that here as well?

Reply:
Yes, because I think there are more people who read this code than conftest.py.